### DISCUSSION TITLE
How to use import.meta.test

### DISCUSSION THREAD

**Discussion by hengshanMWC** [vitest-dev/vitest#1062] [Q&A] [answered]
Which file do you want to import,Or should I write the `d.ts` file?
<img width="451" alt="image" src="https://user-images.githubusercontent.com/45714422/161077658-eea3599a-7415-46e2-8f8c-7006f19b4a15.png">
<img width="451" alt="image" src="https://user-images.githubusercontent.com/45714422/161077658-eea3599a-7415-46e2-8f8c-7006f19b4a15.png">

**Comment by sheremet-va** [comment, marked as answer]
Add it to types in your tsconfig: https://vitest.dev/guide/in-source.html#typescript

**Reply by sheremet-va** [reply]
It's not "test", but "vitest": `import.meta.vitest`

**Reply by hengshanMWC** [reply]
Thank you!
